.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
`node_modules` is a directory, so you need to list it in the `.gitignore` with a slash: `/node_modules`